lib/connectionMiddlewareInit.js

```diff
@@ -2,26 +2,16 @@
 
 const http = require('http');
 const util = require('./util');
-const compose = require('koa-compose');
```

Comment:
There is no need to do the middleware process every connection and every packet event.